# This message type describes a Triangle-Mesh-Geometry which is defined by vertices and and faces.
Header header
geometry_msgs/Point32[] vertices
geometry_msgs/Pose pose
Face[] faces
